### PR TITLE
Update default embedded Ruby version from 2.2 to 2.4.

### PR DESCRIPTION
**Pull request by qingling128**

### CODE REVIEW

File: config/software/td-agent-cleanup.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@
 build do
   block do
     project_name = project.name
-    gem_dir_version = "2.2.0"
+    gem_dir_version = "2.4.3"
```
Comment (igorpeshansky) [inline config/software/td-agent-cleanup.rb]: Should this be `2.4.0`?

File: config/software/td-agent-files.rb [hunk 2]
```diff
@@ -14,7 +14,7 @@
     install_path = project.install_dir # for ERB
     project_name = project.name # for ERB
     project_name_snake = project.name.gsub('-', '_') # for variable names in ERB
-    gem_dir_version = "2.2.0"
+    gem_dir_version = "2.4.3"
```
Comment (igorpeshansky) [inline config/software/td-agent-files.rb]: Should this be `2.4.0`?

File: config/software/td-agent-ui.rb [hunk 3]
```diff
@@ -14,7 +14,7 @@
     project_name = project.name # for ERB
     project_name_snake = project.name.gsub('-', '_') # for variable names in ERB
     project_name_snake_upcase = project_name_snake.upcase
-    gem_dir_version = "2.2.0"
+    gem_dir_version = "2.4.3"
```
Comment (igorpeshansky) [inline config/software/td-agent-ui.rb]: Should this be `2.4.0`?

File: config/software/td.rb [hunk 4]
```diff
@@ -13,7 +13,7 @@
     project_name = project.name # for ERB
     project_name_snake = project.name.gsub('-', '_') # for variable names in ERB
     project_name_snake_upcase = project_name_snake.upcase
-    gem_dir_version = "2.2.0"
+    gem_dir_version = "2.4.3"
```
Comment (igorpeshansky) [inline config/software/td.rb]: Should this be `2.4.0`?

File: config/software/td-agent-cleanup.rb (hunk 1 above)
Comment (dhrupadb) [reply config/software/td-agent-cleanup.rb]: +1 . See #45 for context.

Comment (qingling128) [reply config/software/td-agent-cleanup.rb]: Changed.

File: config/software/td-agent-files.rb (hunk 2 above)
Comment (qingling128) [reply config/software/td-agent-files.rb]: Changed.

File: config/software/td-agent-ui.rb (hunk 3 above)
Comment (qingling128) [reply config/software/td-agent-ui.rb]: Changed.

File: config/software/td.rb (hunk 4 above)
Comment (qingling128) [reply config/software/td.rb]: Changed.

**Pull request merged**